finish/src/test/java/it/io/openliberty/guides/faulttolerance/FaultToleranceTest.java

```diff
@@ -64,92 +41,49 @@ public void setup() {
         client.close();
         response.close();
-        changeSystemProperty(SYSTEM_MAINTENANCE_TRUE, SYSTEM_MAINTENANCE_FALSE);
     }
 
-    @Test
-    public void testSuite() throws InterruptedException {
-        testFallbackForGet();
-        testRetryGettingSystemProperties();
-    }
-
-    //tag::javadoc[]
+    // tag::javadoc[]
     /**
-     * testFallbackForGet - test for checking if the fallback is being called correctly
-     * 1. Return system properties for a hostname when inventory service is available.
-     * 2. Make System service down and get the system properties from inventory service
-     *    when it is down.
-     * 3. Check if system properties for the specific host was returned when the inventory
-     *    service was down by:
-     *    Asserting if the total number of the system properties, when service is up, is
-     *    greater than the total number of the system properties when service is down.
-     * @return {void}
-     * @throws InterruptedException 
+     * testFallbackForGet - test for checking if the fallback is being called
+     * correctly 1. Return system properties for a hostname when inventory
+     * service is available. 2. Make System service down and get the system
+     * properties from inventory service when it is down. 3. Check if system
+     * properties for the specific host was returned when the inventory service
+     * was down by: Asserting if the total number of the system properties, when
+     * service is up, is greater than the total number of the system properties
+     * when service is down.
      */
-    //end::javadoc[]
+    // end::javadoc[]
+    @Test
     public void testFallbackForGet() throws InterruptedException {
-        response = this.getResponse(baseUrl + INVENTORY_LOCALHOST);
-        assertResponse(baseUrl, response);
+        response = TestUtils.getResponse(client,
+                                         TestUtils.INVENTORY_LOCALHOST_URL);
+        assertResponse(TestUtils.baseUrl, response);
         JsonObject obj = response.readEntity(JsonObject.class);
         int propertiesSize = obj.size();
-        changeSystemProperty(SYSTEM_MAINTENANCE_FALSE, SYSTEM_MAINTENANCE_TRUE);
+        TestUtils.changeSystemProperty(TestUtils.SYSTEM_MAINTENANCE_FALSE,
+                                       TestUtils.SYSTEM_MAINTENANCE_TRUE);
         Thread.sleep(3000);
-        response = this.getResponse(baseUrl + INVENTORY_LOCALHOST);
-        assertResponse(baseUrl, response);
+        response = TestUtils.getResponse(client,
+                                         TestUtils.INVENTORY_LOCALHOST_URL);
+        assertResponse(TestUtils.baseUrl, response);
         obj = response.readEntity(JsonObject.class);
         int propertiesSizeFallBack = obj.size();
-        assertTrue("The total number of properties from the @Fallback method is not smaller than the number from the system service, as expected.", propertiesSize > propertiesSizeFallBack);
```

Comment:
@sishida can you help review this message, "The total number of properties from the @Fallback method is not smaller than the number from the system service, as expected."?

Reply:
@evelinec: I somehow missed this! Sorry about that. I think it's good to go...I would remove the last comma before "as expected," but the grammar checker is fine with it either way.